hazpy/report.py

```diff
@@ -1517,41 +1517,75 @@ def buildPremade(self, path):
                 try:
                     economicLoss = results[['block', 'EconLoss', 'geometry']]
                     # convert to GeoDataFrame
-                    breaks = nb(results['EconLoss'], nb_class=4)
-                #    breaks = self.equal_interval(
-                   #     results['EconLoss'].to_list(), 4)
-                    legend_item1 = breaks[0]
-                    legend_item2 = breaks[1]
-                    legend_item3 = breaks[2]
-                    legend_item4 = breaks[3]
-                    legend_item5 = breaks[4]
-                    floodDataDictionary['legend_1'] = (
-                        '$'
-                        + self.abbreviate(legend_item1)
-                        + '-'
-                        + self.abbreviate(legend_item2)
-                    )
-                    floodDataDictionary['legend_2'] = (
-                        '$'
-                        + self.abbreviate(legend_item2)
-                        + '-'
-                        + self.abbreviate(legend_item3)
-                    )
-                    floodDataDictionary['legend_3'] = (
-                        '$'
-                        + self.abbreviate(legend_item3)
-                        + '-'
-                        + self.abbreviate(legend_item4)
-                    )
-                    floodDataDictionary['legend_4'] = (
-                        '$'
-                        + self.abbreviate(legend_item4)
-                        + '-'
-                        + self.abbreviate(legend_item5)
-                    )
+                    print("economicLoss count:", len(economicLoss.index))
```

Comment:
Do we want this print statement in here? There doesn't seem like a need to have it, aside from locally troubleshooting.